backpack/core/derivatives/adaptive_avg_pool_nd.py

```diff
@@ -0,0 +1,112 @@
+"""Implements the derivatives for AdaptiveAvgPool."""
+from typing import List, Tuple, Union
+
+from torch import Size
+from torch.nn import AdaptiveAvgPool1d, AdaptiveAvgPool2d, AdaptiveAvgPool3d
+
+from backpack.core.derivatives.avgpoolnd import AvgPoolNDDerivatives
+
+
+class AdaptiveAvgPoolNDDerivatives(AvgPoolNDDerivatives):
+    """Implements the derivatives for AdaptiveAvgPool."""
+
+    def __init__(self, N: int):
+        """Initialization.
+
+        Args:
+            N: number of dimensions
+        """
+        super(AdaptiveAvgPoolNDDerivatives, self).__init__(N=N)
```

Comment:
This is only calling the parent class constructor -> remove?

Reply:
Removed.